index.html

```diff
@@ -61,14 +61,138 @@ <h4 class="heading">Skills</h4>
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
+        </div>
+        <div class="grid-row-item">
+          <button type="button" class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+      <div class="grid-item proj-2">
+        <div class="grid-row-item">
+          <img src="images/Img-empty.png" alt="Project Image" class="project-image"/>
+        </div>
+        <div class="grid-row-item">
+          <h4 class="project-title">Multi-Post Stories</h4>
+        </div>
+        <div class="shining-star"></div>
+        <div class="grid-row-item">
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
+        </div>
+        <div class="grid-row-item">
+          <button type="button" class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+      <div class="grid-item proj-3">
+        <div class="grid-row-item">
+          <img src="images/Img-empty.png" alt="Project Image" class="project-image"/>
+        </div>
+        <div class="grid-row-item">
+          <h4 class="project-title">Multi-Post Stories</h4>
+        </div>
+        <div class="shining-star"></div>
+        <div class="grid-row-item">
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
+        </div>
+        <div class="grid-row-item">
+          <button type="button" class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+      <div class="grid-item proj-4">
```

Comment:
- [ ] Good job using flexbox to align your project contents 👍🏼 . Your project cards are currently not centralized or centered on the page as they are all aligned to the left. Kindly align the project elements to the center with equal margin to match the Figma design template you chose 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/185467986-eb1e4c9c-6bc7-4b7a-a71c-a9e1a92d53a8.png)  | ![image](https://user-images.githubusercontent.com/63473584/185468291-0a44285b-8d71-4ff7-8489-6c9f2a2bd957.png)  |